src/stories/components/MainMenu/MainMenu.js

```diff
@@ -2,6 +2,7 @@ import './styles.scss';
 
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+// import { NavLink, useHistory, Link } from 'react-router-dom';
```

Comment:
@stenizwahyudi if you're not using this, you should delete the comment.

Reply:
Thank you for catching this, I totally forgot to delete it.